### PR TITLE
PR for the issue #145: Please add redux-devtools-dock-monitor.

### PR DESCRIPTION
**Pull request by standino**
Hi @mxstbr ,

This is the PR for the issue #145: Please add redux-devtools-dock-monitor. I have not figured out how to remove it for the production env. Maybe, you can give some advise on that. Thanks.

Will.

### CODE REVIEW

**Comment by mxstbr** [comment]
Amazing work, thanks so much, the DevTools are amazing! I think we need to bump the dependency to Redux in the `package.json` from `^3.0.5` to `^3.1.0` because the new `createStore()` syntax only works in Redux@+3.1.0.

As for the production env, could you try this:

```JS
let store;
if (module.hot) {
  store = createStoreWithMiddleware(rootReducer, fromJS({}), DevTools.instrument());
} else {
  store = createStoreWithMiddleware(rootReducer, fromJS({}));
}
reduxRouterMiddleware.listenForReplays(store, (state) => state.get('route').location);
```

File: app/DevTools.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
```
Comment (mxstbr) [inline app/DevTools.js]: Nit: semi-colon after all these imports please

File: app/DevTools.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+import { createDevTools } from 'redux-devtools'
+import LogMonitor from 'redux-devtools-log-monitor'
+import DockMonitor from 'redux-devtools-dock-monitor'
+
+export default createDevTools(
+  <DockMonitor toggleVisibilityKey="ctrl-h"
+               changePositionKey="ctrl-x">
+    <LogMonitor />
+  </DockMonitor>
+)
```
Comment (mxstbr) [inline app/DevTools.js]: nit: semi-colon here, empty line at the end of the file

File: app/app.js [hunk 3]
```diff
@@ -69,36 +69,39 @@ if (module.hot) {
...
+            />
+            <Route
+              path="*"
+              getComponent={function get(location, cb) {
+                require.ensure([], (require) => {
+                  cb(null, require('NotFoundPage').default);
+                }, 'NotFoundPage');
+              }}
+            />
+            </Route>
```
Comment (mxstbr) [inline app/app.js]: nit: indentation here is messed up a bit

**Comment by mxstbr** [comment]
How are we looking here @standino, any update on this?

**Comment by mxstbr** [comment]
Could you rebase onto the current state of `v3.0.0` and solve the routes merge conflict? Thanks!

**Comment by standino** [comment]
Just back from vacation, I will work on that. Thanks for your comments.

**Pull request closed without merging**